app/models/edm/provided_cho.rb

```diff
@@ -7,15 +7,18 @@
 module EDM
   class ProvidedCHO
     include Mongoid::Document
+    include AutocompletableModel
     include CampaignValidatableModel
     include RDFModel
     include RemoveBlankAttributes
 
     embedded_in :ore_aggregation, class_name: 'ORE::Aggregation', inverse_of: :edm_aggregatedCHO
 
-    embeds_one :dc_creator, class_name: 'EDM::Agent', inverse_of: :dc_creator_for_edm_providedCHO
     embeds_one :dc_contributor, class_name: 'EDM::Agent', inverse_of: :dc_contributor_for
 
+    embeds_many :dc_subject_agent, class_name: 'EDM::Agent', inverse_of: :dc_subject_agent_for
```

Comment:
semantically it should be: `embeds_many :dc_subject_agents`(plural), this will then need to be updated where it's referenced in the agent class, the form etc.